examples/flutter_gallery/web/index.html

```diff
@@ -18,6 +18,16 @@
   <link rel="manifest" href="/manifest.json">
 </head>
 <body>
+  <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+    <script>
+    if ('serviceWorker' in navigator) {
+      window.addEventListener('load', function () {
+        navigator.serviceWorker.register('/service_worker.js');
```

Comment:
For some reason I feel like we should use a more specific name, such as `flutter_service_worker.js`.

Reply:
Done

Reply:
The comment above still references to `service_worker.js`, just so you know